components/events/summer-school/FaqSection.vue

```diff
@@ -0,0 +1,323 @@
+    url: 'https://www.khanacademy.org/math/precalculus/x9e81a4f98389efdf:matrices/x9e81a4f98389efdf:multiplying-matrices-by-matrices/v/matrix-multiplication-intro',
+    segment: { action: 'events-summer-school > faq > khanacademy-matrix-multiplication-intro' }
+  }
+
+  pythonAndJupyter: TrackedLink = {
+    url: 'https://qiskit.org/textbook/ch-prerequisites/python-and-jupyter-notebooks.html',
+    segment: { action: 'events-summer-school > faq > textbook-python-and-jupyter-notebooks' }
+  }
+
+  linearAlgebra: TrackedLink = {
+    url: 'https://qiskit.org/textbook/ch-prerequisites/linear_algebra.html',
+    segment: { action: 'events-summer-school > faq > qiskit-linear-algebra' }
+  }
+
+  qCEducators: TrackedLink = {
+    url: 'https://quantum-computing.ibm.com/programs/educators',
+    segment: { action: 'events-summer-school > faq > ibm-quantum-educators-program' }
+  }
+
+  mailtoQCEducators: TrackedLink = {
+    url: 'mailto:quantum.education@us.ibm.com',
+    segment: { action: 'events-summer-school > faq > mailto-quantum-educators-program' }
+  }
+
+  twitterQiskit: TrackedLink = {
+    url: 'https://twitter.com/qiskit',
+    segment: { action: 'events-summer-school > faq > qiskit-twitter' }
+  }
+
+  twitterIBMResearch: TrackedLink = {
+    url: 'https://twitter.com/IBMResearch',
+    segment: { action: 'events-summer-school > faq > ibmresearch-twitter' }
+  }
+
+  githubQiskit: TrackedLink = {
+    url: 'https://github.com/Qiskit',
+    segment: { action: 'events-summer-school > faq > qiskit-github' }
+  }
+
+  mediumQiskit: TrackedLink = {
+    url: 'https://medium.com/qiskit',
+    segment: { action: 'events-summer-school > faq > qiskit-medium' }
+  }
+
+  qiskitSubscribe: TrackedLink = {
+    url: 'https://qisk.it/sub',
+    segment: { action: 'events-summer-school > faq > qiskit-youtube-subscribe' }
+  }
+
+  joinqiskitslack: TrackedLink = {
+    url: 'https://ibm.co/joinqiskitslack',
+    segment: { action: 'events-summer-school > faq > qiskit-slack-join' }
+  }
+
+  qiskitEvents: TrackedLink = {
+    url: 'https://qiskit.org/events',
+    segment: { action: 'events-summer-school > faq > qiskit-org-events' }
+  }
+
+  mailtoQiskit: TrackedLink = {
+    url: 'mailto:qiskit.events@us.ibm.com',
+    segment: { action: 'events-summer-school > faq > mailto-qiskit-events' }
+  }
+}
+</script>
+
+<style lang="scss" scoped>
+@import "~/assets/scss/layout/_copy.scss";
+
+.faq-section {
+  // stylelint-disable selector-pseudo-element-no-unknown
+  ::v-deep {
+  // stylelint-enable selector-pseudo-element-no-unknown
+    .bx--accordion__title {
+      color: $text-color;
+    }
+
+    .bx--accordion__heading:hover,
+    .bx--accordion__heading:hover::before {
+      background-color: $background-color-lighter;
+    }
+
+    .bx--accordion__arrow {
+      fill: $text-color;
+    }
+
+    .bx--accordion__content {
+      @extend .copy__paragraph;
+
+      max-width: none;
+      > p + p {
+        margin-top: 1.43em;
```

Comment:
Where does this number come from? We should try and use Carbon's $spacing variables.